files/brews/hub.rb

```diff
@@ -0,0 +1,26 @@
+require 'formula'
+
+class Hub < Formula
+  homepage "http://hub.github.com/"
+
+  if RUBY_PLATFORM =~ /x86_64/
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_amd64_v2.2.0-rc1.tar.gz"
+    sha1 "413ce80d32f6313cef010e0edb1e664e1fff7427"
+  else
+    url "https://github.com/github/hub/releases/download/v2.2.0-rc1/hub_mac_386_v2.2.0-rc1.tar.gz"
+    sha1 "059e6d1759b568fd79bacfc397de199135056ff1"
+  end
+
+  def install
+    bin.install "hub"
+    man1.install Dir["man/*"]
+    bash_completion.install "etc/hub.bash_completion.sh"
+    zsh_completion.install "etc/hub.zsh_completion" => "_hub"
+  end
+
+  test do
+    HOMEBREW_REPOSITORY.cd do
+      assert_equal "bin/brew", shell_output("#{bin}/hub ls-files -- bin").strip
```

Comment:
The testing logic is [inherited from the official `hub` formula](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/hub.rb#L37-L41). [I was the advocator to check version](https://github.com/Homebrew/homebrew/pull/28829#discussion_r12097303) but was told to follow what `hub` does. So it sounds like we're back to checking version?